### DISCUSSION TITLE
Is it possible to copy text from PDF?

### DISCUSSION THREAD

**Discussion by mika-kaki** [wojtekmaj/react-pdf#908] [Q&A] [answered]
Hi! Is it possible to allow the user to copy text from a PDF document that is rendered on a page? E.g. the pdf could include some information that the user needs to enter to a field next to the PDF. I mean, we already use react-pdf, but at least the way we "happen" to use it, does not allow the user to copy text. Anything we could do?

**Comment by wojtekmaj** [comment, marked as answer]
As long as you render text layer (`renderTextLayer` flag, on by default), then you should be able to copy the text from a special text layer rendered on top of an actual PDF file.

You won't be able to copy text from the image layer, regardless of the render mode (`canvas`/`svg`).

**Comment by mika-kaki** [comment]
Great! Thanks for your reply and help. We'll look into this!
